<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>appliedjob</class>
 <widget class="QDialog" name="appliedjob">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_apjob">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_backapjob">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sjoblist">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>View Applied Job</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_scj">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_apmain">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>1251</width>
     <height>591</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>1211</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select desired job id to see results</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_view">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>1211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_jn">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Job  ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_jn_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>JobName</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_jn_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Employer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_jobid">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_jobname">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>421</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_employer1">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>90</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_jn_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_status">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>150</y>
       <width>821</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_jn_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_username">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>120</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_vieww">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_test1">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>380</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_test2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>380</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
